config/application.rb

```diff
@@ -32,6 +32,12 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Active Job Configurations
+    if ENV['REDIS_HOST']
+      config.active_job.queue_adapter = :sidekiq
+      # config.active_job.queue_name_prefix = "scholarsphere_production"
```

Comment:
@whereismyjetpack do we need to have a separate queue name for each environment (dev, test, prod) or should this be configured per server environment (qa, stage, production) via ENV? Also, the `else` should be `:async` so we can have a queue adapter that mimics services like sidekiq without having to explicitly be one.

Reply:
Another thought, if this is per server env. should it be the domain of RedisConfig? Don't know...

Reply:
I think the queue name prefix should probably be part of application in the SIDEKIQ_ namespace. could be a single env var..